dask/array/tests/test_array_core.py

```diff
@@ -722,6 +722,16 @@ def test_roll(chunks, shift, axis):
     assert_eq(np.roll(x, shift, axis), da.roll(a, shift, axis))
 
 
+@pytest.mark.parametrize('chunks', [(4, 6), (2, 6)])
+@pytest.mark.parametrize('shift', [(3, 9), (7, 2)])
+@pytest.mark.parametrize('axis', [(0, 1), (1, 0)])
+def test_roll_multiple(chunks, shift, axis):
+    x = np.random.randint(10, size=(4, 6))
+    a = from_array(x, chunks=chunks)
+
+    assert_eq(np.roll(x, shift, axis), da.roll(a, shift, axis))
```

Comment:
Have now consolidated them. Caught a bug in the process. So definitely worth the effort.